src/index.js

```diff
@@ -27,7 +27,7 @@ class GifPlayerContainer extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      playing: false,
+      playing: !!Boolean(props.autoPlay),
```

Comment:
The `!!` is redundant here (`Boolean(...)` already casts to boolean) so you can remove it.

Reply:
You are right.